smdebug/core/hook.py

```diff
@@ -460,6 +489,8 @@ def _cleanup(self):
             self.metrics_writer.close()
 
         training_has_ended(self.out_dir)
+        if self.first_process is True:
+            remove_claim_file(self.out_dir)
```

Comment:
is this reqd? 
Is there any side effect if we don't remove this?

Reply:
There is no side effect if we do not remove this.
The code has been tested without the removal of this file as well.